packages/hardhat/deploy/00_deploy_your_contract.ts

```diff
@@ -1,26 +1,34 @@
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 import { DeployFunction } from "hardhat-deploy/types";
 
+/**
+ * Deploys a contract named "YourContract" using the deployer account and
+ * constructor arguments set to the deployer address
+ *
+ * @param hre HardhatRuntimeEnvironment object.
+ */
 const deployYourContract: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployer } = await hre.getNamedAccounts();
   const { deploy } = hre.deployments;
 
   await deploy("YourContract", {
     from: deployer,
-    // Constructor args.
-    // Use your frontend address
+    // By default the deployer account will be one that comes with Hardhat, which anyone can access.
```

Comment:
Nitpick: my IDE is telling me this

![image](https://user-images.githubusercontent.com/2486142/214535088-f95b2315-92cb-4a36-9a75-0d8531d742bf.png)

Reply:
grammar :D